pkgs/top-level/aliases.nix

```
@@ -301,6 +301,7 @@ mapAliases ({
   cloud-print-connector = throw "Google Cloudprint is officially discontinued since Jan 2021, more info https://support.google.com/chrome/a/answer/9633006";
   cquery = throw "cquery has been removed because it is abandoned by upstream. Consider switching to clangd or ccls instead"; # Added 2020-06-15
   cups-googlecloudprint = throw "Google Cloudprint is officially discontinued since Jan 2021, more info https://support.google.com/chrome/a/answer/9633006";
+  cups-kyodialog3 = lib.warn "cups-kyodialog3 has been renamed/replaced by cups-kyodialog" cups-kyodialog; # Added 2022-11-12
```

Why not an actual alias?
```suggestion
  cups-kyodialog3 = cups-kyodialog; # Added 2022-11-12
```

---

Agreed; warning removed. It would have been the only entry in aliases.nix that triggered a warning.